index.rst

```diff
@@ -1,51 +1,628 @@
-..
-  Technote content.
+.. vim: ts=3:sts=3
 
-  See https://developer.lsst.io/docs/rst_styleguide.html
-  for a guide to reStructuredText writing.
+:tocdepth: 1
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+.. note::
 
-  Use the following syntax for sections:
+   This material was transcribed from the `Winter 2014 CameraGeom Design`_, originally written by Russell Owen & Simon Krughoff.
+   See also material on `the representation of a camera`_ and the `CameraGeom documentation`_.
 
-  Sections
-  ========
+.. _Winter 2014 CameraGeom Design: https://dev.lsstcorp.org/trac/wiki/Winter2014/Design/CameraGeomDesign
```

Comment:
I need to have that first coffee (and I need to read the actual title of the pull request).... Reading the rendered version rather than PR version would help me.